python/cudf/cudf_pandas_tests/third_party_integration_tests/dependencies.yaml

```diff
@@ -244,6 +251,16 @@ dependencies:
           - pip
           - pip:
             - xgboost>=2.0.1
+  test_catboost:
+    common:
+      - output_types: conda
+        packages:
+          - numpy
+          - scipy
+          - scikit-learn
+          - pip
+          - pip:
+            - catboost
```

Comment:
Do we need to install this via pip? Appears catboost is on conda forge https://catboost.ai/en/docs/installation/python-installation-method-conda-install

Reply:
Please use conda if possible.